en/services/parachute.md

```diff
@@ -0,0 +1,165 @@
+| 6   | 64       | `PARACHUTE_ERROR_FLAGS_POD_CONNECTION_ERROR`   | There is an error with the parachute pod.                                                                                  |
+| 7   | 128      | `PARACHUTE_ERROR_FLAGS_EFTS_DIAGNOSE`          | Parachute Electrical Flight Termination System (EFTS) diagnosis failed.                                                    |
+| 8   | 256      | `PARACHUTE_ERROR_FLAGS_CHARGING_FAILED`        | Parachute module charging failed.                                                                                          |
+| 9   | 512      | `PARACHUTE_ERROR_FLAGS_EXTERNAL_POWER_ERROR`   | There is an error with the parachute external power source.                                                                |
+| 10  | 1024     | `PARACHUTE_ERROR_FLAGS_GS_CONNECTION_ERROR`    | There is an error with the connection between parachute and Ground Station (GS).                                           |
+| 11  | 2048     | `PARACHUTE_ERROR_FLAGS_GPS_ERROR`              | There is an error with the parachute's GPS.                                                                                |
+| 12  | 4096     | `PARACHUTE_ERROR_FLAGS_SUBSYSTEM_CONNECTION_ERROR` | There is an error with the connection between parachute and subsystem (e.g. remote controller, expansion board, etc.). |
+| 13  | 8192     | `PARACHUTE_ERROR_FLAGS_SUBSYSTEM_FW_ERROR`     | There is an error with the parachute subsystem firmware (e.g. wrong firmware version).                                     |
+| 14  | 16384    | `PARACHUTE_ERROR_FLAGS_RESERVED_1`             | Reserved for future use.                                                                                                   |
+| 15  | 32768    | `PARACHUTE_ERROR_FLAGS_RESERVED_2`             | Reserved for future use.                                                                                                   |
+| 16  | 65536    | `PARACHUTE_ERROR_FLAGS_LOGGING_ERROR`          | There is an error with the parachute's internal logging system.                                                            |
+| 17  | 131072   | `PARACHUTE_ERROR_FLAGS_MODULE_RETIRED`         | This parachute module is retired (i.e. too many deployments).                                                              |
+| 18  | 262144   | `PARACHUTE_ERROR_FLAGS_GLOW_WIRE_ERROR`        | There is an error with the parachute glow wire.                                                                            |
+| 19  | 524288   | `PARACHUTE_ERROR_FLAGS_OFFBOARD_CONNECTION_ERROR` | There is an error with the MAVLink connection between parachute and offboard computer.                                 |
+| 20  | 1048576  | `PARACHUTE_ERROR_FLAGS_IMU_CALIBRATION_ERROR`  | Parachute's internal IMU calibration failed.                                                                               |
+                                                                    |
```

Comment:
this looks to be extra and not actually part of the table

Reply:
yes good catch i forgot to double check the preview